src/handlers.rs

```diff
@@ -29,9 +29,7 @@ pub extern "C" fn handle_code_fetch(
 
     println!(
         "handle_code_fetch. cmd_id = {} module_specifier = {} containing_file = {}",
-        cmd_id,
-        module_specifier,
-        containing_file
+        cmd_id, module_specifier, containing_file
```

Comment:
Unrelated change (it should have been formatted elsewhere, but it would be good if you left it out)

Reply:
gotcha, sorry about that. I'll leave this in for now yeah?